lib/src/utils/crypto/native.dart

```diff
@@ -155,13 +160,15 @@ abstract class Cipher {
     final initialVectorDecoded = base64decodeUnpadded(base64.normalize(encryptedFileInfo.initialVector));
     final algorithm = getAlg(keyDecoded.length * 8);
 
-    final memSize = sizeOf<IntPtr>() + keyDecoded.length + initialVectorDecoded.length + maxHashSize;
+    final memSize = sizeOf<IntPtr>() + keyDecoded.length + initialVectorDecoded.length + maxHashSize + 1;
```

Comment:
why `+`? magic number

Reply:
it's for memory of the value of the length for hash value. 
<img width="586" alt="Screenshot 2023-07-31 at 09 11 15" src="https://github.com/linagora/matrix-dart-sdk/assets/43041967/1e918076-85f4-4a38-868c-622f7cc6ab6d">